wp4h_gxp_c_fhir_details.md

```diff
@@ -1,4 +1,4 @@
----
+<!-- ---
```

Comment:
It looks like you modified this to an HTML-style comment, but I don't think that's supposed to happen.

For comparison, here's an excerpt from a file in the Watson Text-to-speech GS doc:

![image](https://user-images.githubusercontent.com/23705736/43025295-2e24acbe-8c3f-11e8-8e2f-504bfdda33c2.png)

Reply:
I think you only need HTML-style comments if you're adding them in the context of an HTML element (like a table).

Reply:
We don't want to introduce formatting anomalies, so please revert changes that you made to comment formatting back to their original format, unless you know that they have to be in HTML format.

Reply:
I just created a new branch called "test.' I'm going to reject/close this PR and ask that after you make your changes, you submit your PR to that branch.